### ISSUE TITLE
Error opening compmgmt.msc from cmd

### ISSUE THREAD

**Issue by rignatev** [gerardog/gsudo#29]
Hi! Nice work!

I have the error when launching compmgmt.msc from cmd.exe with gsudo v0.7.
In powershell all is ok.

```
Microsoft Windows [Version 10.0.18363.720]
(c) Корпорация Майкрософт (Microsoft Corporation), 2019. Все права защищены.

C:\Users\***>sudo compmgmt.msc
Error: System.ComponentModel.Win32Exception (0x80004005): Операция успешно завершена
   в gsudo.Helpers.ProcessFactory.CreateProcessAsUserWithFlags(String lpApplicationName, String args, CreateProcessFlags dwCreationFlags, PROCESS_INFORMATION& pInfo)
   в gsudo.ProcessRenderers.TokenSwitchRenderer..ctor(Connection connection, ElevationRequest elevationRequest)
   в gsudo.Commands.RunCommand.<RunUsingSingleUseElevation>d__7.MoveNext()
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в gsudo.Commands.RunCommand.<Execute>d__5.MoveNext()
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в gsudo.Program.<Start>d__1.MoveNext()
```

**Comment by gerardog** [comment]
Thanks for reporting. Starting non-executables with valid file associations is failing.
As a workaround use `sudo start compmgmt.msc`

**Comment by gerardog** [comment]
The fix was released in [v0.7.1](https://github.com/gerardog/gsudo/releases/tag/v0.7.1).
But there now one new related issue: `gsudo compmgmt` will wait until the MMC window is closed before releasing the console. Far from ideal. But it was that, or rellying on Windows PowerShell to launch it (which is kinda slow), or manually resolving the file association (somethoing I could do on a future release...)  
So, please use `gsudo start compmgmt` to avoid locking the console.
